internal/injector/aspect/advice/call.go

```diff
@@ -108,7 +108,9 @@ func (a *appendArgs) AsCode() jen.Code {
 
 func (a *appendArgs) AddedImports() []string {
 	imports := make([]string, 0, len(a.templates)+1)
-	imports = append(imports, a.typeName.ImportPath())
+	if argTypeImportPath := a.typeName.ImportPath(); argTypeImportPath != "" {
+		imports = append(imports, argTypeImportPath)
+	}
```

Comment:
Codecov says that this is not covered. Can you either write a test or a comment?

Reply:
That's odd because I added this code path to fix a bug... need to check this one!

Reply:
I added a specific unit test for that ^^